### ISSUE TITLE
Warning upon installation.

### ISSUE THREAD

**Issue by codeviking** [ganderzz/react-scroll-to#99]
**Describe the bug**
Installation yields a warning about the `core-js` version:

```
warning react-scroll-to > create-react-context > fbjs > core-js@1.2.7: core-js@<2.6.5 is no longer maintained. Please, upgrade to core-js@3 or at least to actual version of core-js@2.
```

**To Reproduce**
1. `yarn add react-scroll-to`

**Expected behavior**
That it installs without a warning about an unmaintained dependency.

**Additional context**
Ultimately this is a dependency of `create-react-context`.  Given the context API has stabilized, maybe you could just port the library to use the official context API instead of the polyfilled one?

**Comment by ganderzz** [comment]
@codeviking Thanks for bringing this up. I agree, with context being a few versions behind us now I think it's fair to get rid of the polyfill.

Open to any PRs on this. 👍

**Comment by ganderzz** [comment]
This has been merged, and will be released with v3.0.0 !
